actions/admin_actions.jsx

```diff
@@ -173,6 +173,23 @@ export async function uploadPrivateSamlCertificate(file, success, error) {
     }
 }
 
+export async function uploadPublicLdapCertificate(file, success, error) {
```

Comment:
Actions using callbacks are something we've been trying to replace since we've supported promises for a while and since they require directly accessing the global instance of the redux store. We should be calling the mattermost-redux version of those options if possible.

Reply:
Oh, I just realized every action in `admin_defintions.jsx` uses these callback-based actions. If they're required there, I'd rather replace them, but that seems like it might be out of scope for this PR.

Reply:
@hmhealey we'll make a ticket to get rid of all these callbacks in a separate PR